extensions/schema-compare/src/utils.ts

```diff
@@ -50,7 +52,15 @@ export function getEndpointName(endpoint: mssql.SchemaCompareEndpointInfo): stri
 		if (!endpoint.serverName && endpoint.connectionDetails) {
 			endpoint.serverName = endpoint.connectionDetails['serverName'];
 		}
-		return `${endpoint.serverName}.${endpoint.databaseName}`;
+		if (!endpoint.databaseName && endpoint.connectionDetails) {
+			endpoint.databaseName = endpoint.connectionDetails['databaseName'];
+		}
+		if (endpoint.serverName && endpoint.databaseName) {
+			return `${endpoint.serverName}.${endpoint.databaseName}`;
+		} else {
+			return ' ';
```

Comment:
You don't want to return anything if we just have a database or just have a server?

Reply:
In the ideal scenario, SCMP will have both values unless manually changed. And if manually modified with inappropriate values, it doesn't make sense to show any details, since we anyways won't be able to compare anything then.